Makefile

```diff
@@ -28,17 +25,24 @@ ifeq (${USE_CACHE}, yes)
 	MELANGE_OPTS += --cache-source ${CACHE_DIR}
 endif
 
-ifeq (${BUILDWORLD}, no)
-MELANGE_OPTS += -k ${WOLFI_SIGNING_PUBKEY}
-MELANGE_OPTS += -r ${WOLFI_PROD}
-endif
-
 # The list of packages to be built. The order matters.
 # wolfictl determines the list and order
 # set only to be called when needed, so make can be instant to run
 # when it is not
 PKGLISTCMD ?= $(WOLFICTL) text --dir . --type name --pipeline-dir=./pipelines/
 
+ifeq (${BOOTSTRAP}, yes)
+	MELANGE_OPTS += -k https://packages.wolfi.dev/bootstrap/stage3/wolfi-signing.rsa.pub
+	MELANGE_OPTS += -r https://packages.wolfi.dev/bootstrap/stage3
+	PKGLISTCMD += -k https://packages.wolfi.dev/bootstrap/stage3/wolfi-signing.rsa.pub
+	PKGLISTCMD += -r https://packages.wolfi.dev/bootstrap/stage3
+else
+	MELANGE_OPTS += -k https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+	MELANGE_OPTS += -r https://packages.wolfi.dev/os
+	PKGLISTCMD += -k https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+	PKGLISTCMD += -r https://packages.wolfi.dev/os
```

Comment:
Very likely a silly question but I am curious: since these values are all repeated twice, is there a specific reason for not keeping the variable approach?

Reply:
Not silly at all! It's just a stylistic choice, and I don't feel strongly about it either way. I'll flip it the other way, that seems less error prone.